.gitignore

```diff
@@ -1,6 +1,12 @@
+# Mac OS X files
+.DS_Store
+
 # IDE commons
 .idea
+.vscode/*
+.history/
```

Comment:
What is this for?

Reply:
Thats from vscode. Some of my colleagues are using that for go coding too.